src/packages/database/index.js

```diff
@@ -43,7 +45,7 @@ class Database {
   }
 
   modelFor(type: string): Class<Model> {
-    const model = this.models.get(type);
+    const model = this.models.get(normalizeModelName(type));
```

Comment:
Before attempting to resolve a model the string is first normalized to ensure it is formatted the way model's are stored in `Database#models`.